website/content/en/docs/setup/going-to-prod/high-availability.md

```diff
@@ -0,0 +1,184 @@
+short: HA
+weight: 3
+---
+
+## Failure Modes
+
+Before making a system highly available, you must understand the various ways it can fail.
+
+### Hardware Failures
+
+#### Disk Failure
+
+Disk failures are treated as [process failures](#vector-process-failure) since Vector will exit if the disk is unavailable when Vector needs it. Vector will fail to start if the disk is unavailable at boot time, and Vector will exit if the disk is unavailable at runtime.
+
+Vector does not need a disk during runtime unless you’re using a component that requires it, such as disk buffers or the `file` source and sink.
+
+#### Node Failure
+
+Mitigate individual node failures by [distributing](/docs/setup/going-to-prod/architecting/) Vector, and its load balancer, across multiple nodes. Front the Vector nodes with a load balancer that will failover when a Vector node becomes unreachable. You should have enough capacity to distribute the tolerated node failure across your available nodes. A good rule of thumb is that no single node should process more than 33% of your data. Finally, automated self-healing should detect and replace unhealthy nodes.
+
+![Node failure](/img/going-to-prod/ha/node-failure.png)
+
+#### Data Center Failure
+
+Mitigate data center failures by deploying Vector across multiple Availability Zones. For uninterrupted failover, you should have enough capacity in other Availability Zones to handle the loss of any single Availability Zone. Vector’s high-performance design goal exists to make it economically possible to over-provision in this manner to handle failover seamlessly.
+
+![Datacenter failure](/img/going-to-prod/ha/dc-failure.png)
+
+### Software Failures
+
+#### Vector Process Failure
+
+Mitigate Vector process failures similarly to [node failures](#node-failure): Vector’s load balancer will failover when a Vector process becomes unreachable. Automated platform-level self-healing restarts the process or replaces the node. This is achieved with a platform-level supervisor, such as a controller in Kubernetes.
+
+![Process failure](/img/going-to-prod/ha/process-failure.png)
+
+#### Data Receive Failure
+
+Mitigate data receive failures with Vector’s [end-to-end acknowledgements feature](/docs/about/under-the-hood/architecture/end-to-end-acknowledgements/). When enabled, Vector only responds to the clients when data durably persists. Such as writing the data to the disk buffer or a downstream service.
+
+![Failure](/img/going-to-prod/ha/receive-failure.png)
+
+This feature should be paired with disk buffers or streaming sinks to ensure acknowledgements are timely.
+
+#### Data Processing Failure
+
+Mitigate data processing failures with Vector’s dropped events routing. Any component that can fail processing drops events, and comes with a `dropped` output channel that routes dropped events through another pipeline. This flexible design allows the routing of dropped events like any other event.
```

Comment:
Only the `remap` transform has `dropped` right now. I might add a note that expanding this is in work-in-progress.